Simplenote/src/main/java/com/automattic/simplenote/NoteEditorActivity.java

```diff
@@ -143,6 +163,12 @@ public void onPageScrollStateChanged(int state) {
 
             isMarkdownEnabled = savedInstanceState.getBoolean(NoteEditorFragment.ARG_MARKDOWN_ENABLED);
             isPreviewEnabled = savedInstanceState.getBoolean(NoteEditorFragment.ARG_PREVIEW_ENABLED);
+            mSearchMatchIndex = savedInstanceState.getInt(STATE_MATCHES_INDEX, 0);
+            mSearchMatchIndexes = savedInstanceState.getIntArray(STATE_MATCHES_LOCATIONS);
+        }
+
+        if (intent.hasExtra(NoteEditorFragment.ARG_MATCH_OFFSETS)) {
```

Comment:
is it right to assume that these were already present, coming from the fulltext-search performed already when searching?

Reply:
yeah I was just making sure we weren't looking for this to be newly added as a part of this PR which might have been missed vs. it already existed in the codebase but we weren't using it

Reply:
Yep, it was already there, but just not used for this purpose.